features/git-town-sync/folder_does_not_exist_on_main_branch/no_conflict.feature

```diff
@@ -49,20 +49,3 @@ Feature: git town-sync: syncing inside a folder that doesn't exist on the main b
       | other-feature   | local and remote | other feature commit                     | file2            |
       |                 |                  | main commit                              | main_file        |
       |                 |                  | Merge branch 'main' into other-feature   |                  |
-
-
-  Scenario: undo
```

Comment:
I don't think we need to implement undo for sync. Its not useful based on the commands it runs.